nca/NetworkConfig/LiveSim/ingress_controller/ingress_controller.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ingress-nginx
```

Comment:
add a namespace, such as ingress-controller-ns,  so that it does not overlap with default namespace.

Reply:
done

Reply:
I meant that the namespace should be added to the service and pod resources